src/web/client/common/remoteFetchProvider.ts

```diff
@@ -49,10 +49,11 @@ export async function fetchData(accessToken: string, entity: string, entityId: s
     }
 }
 
-function createContentFiles(result: string, entity: string, queryParamsMap: any, entitiesSchemaMap: any, languageIdCodeMap: any, portalsFS: PortalsFS, dataverseOrgUrl: string, accessToken: string, entityId: string) {
+function createContentFiles(result: string, entity: string, queryParamsMap: any, entitiesSchemaMap: any, languageIdCodeMap: any, portalsFS: PortalsFS, dataverseOrgUrl: string, accessToken: string, entityId: string, websiteIdToLanguage: any) {
```

Comment:
I hope the types are addressed in the next PR?

Reply:
yes its addressed in the lint fixes pr